docs/install/docker-plugin.rst

```diff
@@ -0,0 +1,74 @@
+
+========================================
+Installing the Flocker Plugin for Docker
+========================================
+
+Before installing the Flocker plugin for Docker, you will need to have installed Flocker on some nodes, using the instructions in the previous topics.
```

Comment:
This prose can probably be expanded and clarified a bit more and it might also be a good idea to link again to the node installation instructions where we say "installed Flocker on some nodes".